packages/relay-experimental/loadQuery.js

```diff
@@ -194,13 +194,6 @@ function loadQuery<TQuery: OperationType, TEnvironmentProviderOptions>(
     const request = getRequest(graphQlTaggedNode);
     params = request.params;
 
```

Comment:
@zth, @josephsavona I think it would be better to manage the moduleId using the id for persisted queries and the cacheID for non-persisted queries instead of removing the check.

const moduleId = params.cacheID ? params.cacheID : params.id;